prac_06/guitar.py

```diff
@@ -0,0 +1,23 @@
+
+
+YEAR = 2020
+
+class Guitar:
+    """guitar calss, stores year, name and cost"""
+
+    def __init__(self, name, year, cost):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return "{} ({}): ${}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        age = YEAR - self.year
+        return age
```

Comment:
Can alternatively fetch the current year so that it is updated automatically using datetime module ie    

def get_age(self):
        from datetime import datetime
        # Reads current year (Used to check if a guitar is vintage)
        current_year = datetime.now().year
        guitar_age = current_year - self.year
        return guitar_age

Reply:
This is cool. I did not know about this but makes sense that python would have an inbuilt date and time thing. Thanks for the feedback